docs/hoon/twig/ket-cast/lus-like.md

```diff
@@ -16,11 +16,13 @@ within the span of `q`.  Otherwise, `nest-fail`.
 
 Regular: *2-fixed*.
 
-Irregular: `\`_foo\`bar` is `:like(foo bar)`.
-
 ## Examples
 
 ```
-~zod:dojo> :like('text' 97)
+~zod:dojo> :like('text' %a)
 'a'
+~zod:dojo> :like('text' 97)
+nest-fail
 ```
```

Comment:
This error is actually caused by a minor flaw in the language.  `@t`97
works but ^-(@t 97) doesn't, because `` also has the magic power of casting
across auras.

Obviously this difference between regular and irregular syntax is not at
all the Hoon way -- this should be done better.

On Mon, Jun 6, 2016 at 5:16 PM, Joseph Bryan <notifications@github.com>
wrote:

> In docs/hoon/twig/ket-cast/lus-like.md
> <https://github.com/urbit/docs/pull/25#discussion_r65992046>:
>
> >  'a'
> > +~zod:dojo> :like('text' 97)
> > +nest-fail
> >  ```
>
> That example fails as written; not sure if that should be documented, or
> just removed...
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/urbit/docs/pull/25/files/3924744aa715a8c2b20bd53910de602d17ad425e#r65992046>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALyAXZSeJmi0HNgewoflnkP70C48f1gks5qJLhZgaJpZM4IvcDK>
> .
>